app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/App.java

```diff
@@ -13,6 +14,8 @@ public static void main(String[] args) {
 
         app.get("/health", ctx -> ctx.result("Operational"));
 
+        ApplicationContext.register(Slf4jLogger.class, Slf4jLogger.getLogger());
```

Comment:
```suggestion
        ApplicationContext.register(Logger.class, Slf4jLogger.getLogger());
```
We want to register the `Slf4jLogger` under just the `Logger` class.

Reply:
Good catch, having the parent is the way to go. I'm still working on having the implementation in the ApplicationContext